app-web/src/utils/algolia.js

```diff
@@ -101,19 +105,23 @@ export const reduceNodesForIndex = nodes => {
     } = node;
     let fields,
       content,
+      persona,
       id = node.id;
 
     if (type === 'GithubRaw') {
       fields = node.childMarkdownRemark.fields;
       content = node.childMarkdownRemark.content;
+      persona = node.childMarkdownRemark.frontmatter.personas;
```

Comment:
isn't personas a field

Reply:
it is a field for Devhub Siphon Nodes, however it exists within the frontmatter object for github raw nodes

Reply:
lets make this a field for github raw nodes.

Reply:
can be done in another pr